packages/edit-site/src/components/page-library/style.scss

```diff
@@ -0,0 +1,91 @@
+.edit-site-library {
+	background: rgba(0, 0, 0, 0.05);
+	margin: $header-height 0 0;
+	.components-text {
+		color: $gray-600;
+	}
+
+	.components-heading {
+		color: $white;
+	}
+
+	@include break-medium {
+		margin: 0;
+	}
+}
+
+.edit-site-library__grid {
+	column-gap: $grid-unit-30;
+	@include break-large() {
+		column-count: 2;
+	}
+
+	// Small top padding required to avoid cutting off the visible outline
+	// when hovering items.
+	padding-top: $border-width-focus-fallback;
+	margin-bottom: $grid-unit-40;
+
+	.edit-site-library__pattern {
+		break-inside: avoid-column;
+		display: flex;
+		flex-direction: column;
+		margin-bottom: $grid-unit-60;
+
+		// Temp styles for testing - delete asap
+		.edit-site-library__preview {
+			border-radius: $radius-block-ui;
+			cursor: pointer;
+			overflow: hidden;
+
+			&.is-inactive {
+				cursor: default;
+			}
+		}
+
+		.edit-site-library__footer,
+		.edit-site-library__button {
+			color: $gray-600;
+		}
+		// END: temp styles for testing.
+	}
+
+	.edit-site-library__preview {
+		flex: 1;
+		margin-bottom: $grid-unit-20;
+	}
+}
+
+// The increased specificity here is to overcome component styles
+// without relying on internal component class names.
+.edit-site-library__search {
+	&#{&} input[type="search"] {
+		background: $gray-800;
+		color: $gray-100;
+
+		&:focus {
+			background: $gray-800;
+		}
+	}
+
+	svg {
+		fill: $gray-600;
+	}
+}
+
+.edit-site-library__pattern-title {
+	color: $gray-600;
+
+	svg {
+		border-radius: $grid-unit-05;
+		background: var(--wp-block-synced-color);
+		fill: $white;
+	}
+}
+
+.edit-site-library__no-results {
+	color: $gray-600;
+	// height: 100%,
```

Comment:
Can we delete these?

Reply:
Done: https://github.com/WordPress/gutenberg/pull/51078/commits/43b4bbefd8ae1a056d80931bbdc0c0438191f304